clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1510,7 +1510,7 @@ class raw2trace_t {
     bool
```

Comment:
> raw2trace_t::handle_rseq_abort_marker() uses different names of the last parameter. 

Mention that this is from PR #7058 (otherwise this sounds like a mismatch that has been there for years or sthg).